### DISCUSSION TITLE
Where to find Python Client Documentation

### DISCUSSION THREAD

**Discussion by bronzy295843** [ravendb/ravendb#14378] [Q&A] [answered]
I have been trying to look up the Python API's documentation and a lot of the stuff is missing. I'm new to Ravendb and am working with its python client to work on a college project. I'm missing even the most basic instructions regarding the document store and session management. I would highly appreciate if someone could point me in the right direction.To clarify, I'm using version 5.3 of RavenDB

**Comment by ml054** [comment, marked as answer]
Hi @bronzy295843 

Currently, we're working on a v5.2 client with a totally revamped API.
We'll no longer support the v5.0 when it'll be ready.
The v5.2 client is in the beta phase, available here https://pypi.org/project/ravendb/5.2.0b1 ✌️


Since this rewrite is in progress we don't have updated documentation yet. During rewrite our goal is to have similar API in all clients for consistent development experience (but having code smell in mind). 

Please compare those tests:
Java: https://github.com/ravendb/ravendb-jvm-client/blob/v5.2/src/test/java/net/ravendb/client/test/client/QueryTest.java#L229
Python: https://github.com/ravendb/ravendb-python-client/blob/v5.2/ravendb/tests/jvm_migrated_tests/query_tests/test_query.py#L135
NodeJs: https://github.com/ravendb/ravendb-nodejs-client/blob/v5.2/test/Ported/QueryTest.ts#L148

Couple of links that might be helpful before we finish our docs for python.

- http://ravendb.net/docs - documentation page - it contains examples in different languages but in Python you usually need to convert method name from Pascal to snake_case :)
- https://github.com/ravendb/ravendb-python-client/tree/v5.2/ravendb/tests/jvm_migrated_tests - test cases might be great source of information about real world usage of given function. 

I hope it helps.
